packages/node_modules/nav-frontend-skjema/src/fnr-input.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import Input, { NavFrontendInputProps } from './input';
+import { erGyldigFodselsnummer } from './fnr-utils';
```

Comment:
Ser at NAV allerede har en NPM-pakke for dette her: https://github.com/navikt/fnrvalidator. Virker som at den [nettopp ble utbedret](https://github.com/navikt/fnrvalidator/pull/30) med skikkelig støtte for D-nummer også (med bidrag fra DSB no less!).

Da tenker jeg det er lurt om vi heller bruker den her 🙂

Reply:
Supert! Veldig bra innspill! Har tatt den i bruk!